### PR TITLE
Fix local cluster manifests

### PR DESCRIPTION
**Pull request by phoracek**
`make cluster-sync` and `make cluster-clean` has been broken with
b3c6fcc. In this patch, we use newly introduced `make manifests`
to generate manifests with local cluster registry and then use
them to deploy the operator.

On top of that, `WATCH_NAMESPACE` has been changed from `""` to operator's
own namespace. That causes controllers to reconcile only objects from
our namespace and therefore ignore components deployed to their respective
namespaces. This bug prevented NetworkAddonsConfig.Status field to be
updated with state of deployed DaemonSets and Deployments.

### CODE REVIEW

**Comment by phoracek** [comment]
@booxter @SchSeba could you please take a look?

**Comment by ovirt-infra** [comment]
All tests passed

**Comment by phoracek** [comment]
@SchSeba  @booxter ready to be reviewed now, master is saved!

**Comment by ovirt-infra** [comment]
All tests passed

File: cluster/clean.sh [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 echo 'Cleaning up ...'
 
-./cluster/kubectl.sh delete --ignore-not-found -f deploy/cluster-network-addons-operator_01_crd.yaml
-./cluster/kubectl.sh delete --ignore-not-found -f deploy/cluster-network-addons-operator_02_rbac.yaml
-./cluster/kubectl.sh delete --ignore-not-found -f deploy/cluster-network-addons-operator_00_namespace.yaml
+./cluster/kubectl.sh delete --ignore-not-found -f _out/namespace.yaml
```
Comment (SchSeba) [inline cluster/clean.sh]: I think we need to first remove the network-addons-config maybe wait so time for all the related object (multus, macpool, linux bridge, sriov) to be remove then the operator and then the namespace?

File: deploy/olm-catalog/cluster-network-addons/0.0.0/cluster-network-addons-operator.v0.0.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -169,9 +169,6 @@ spec:
                       fieldRef:
                         fieldPath: metadata.name
                   - name: WATCH_NAMESPACE
-                    valueFrom:
```
Comment (SchSeba) [inline deploy/olm-catalog/cluster-network-addons/0.0.0/cluster-network-addons-operator.v0.0.0.clusterserviceversion.yaml]: there is no value here it's fine?

File: cluster/clean.sh (hunk 1 above)
Comment (phoracek) [reply cluster/clean.sh]: You are not wrong. This functionality is not implemented yet (we need to add finalizer that will wait with removal of NetworkAddonsConfig until all components are removed). However, it is not a part of this PR.

File: deploy/olm-catalog/cluster-network-addons/0.0.0/cluster-network-addons-operator.v0.0.0.clusterserviceversion.yaml (hunk 2 above)
Comment (phoracek) [reply deploy/olm-catalog/cluster-network-addons/0.0.0/cluster-network-addons-operator.v0.0.0.clusterserviceversion.yaml]: AFAIK it is. Value is of type `string` and if no value is provided it defaults to `""`.

File: cluster/clean.sh (hunk 1 above)
Comment (SchSeba) [reply cluster/clean.sh]: its ok if we don't wait on this PR but this way we remove the namespace(and the operator inside of it) I think for now we can just remove the CR wait for like 5-10 second so the operator can delete the render manifest and then remove the operator without waiting for all the components to be removed and implement the finalizer in a separate PR

Comment (phoracek) [reply cluster/clean.sh]: I changed it so order of manifest removals is the same as it was before this PR, hence there is no change related to this PR other than changing location of those manifests.

Synchronization is not a top priority now.

**Comment by ovirt-infra** [comment]
All tests passed

**Pull request merged**